pipelines/release-images-cf-deployment/tasks/build_release.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+function build_release() {
+  cf_version="${1}"
```

Comment:
I'm not a fan of braces. They don't protect and the assingments don't do variable expansion.
On the other side if this function had 10 parameters we would apparently need them for `"${10}"` onward... 
Well, you decide :)

Reply:
You are right. It's just a habit to put braces everywhere, so I don't miss them when they are required. I'll leave as is since it has no harm other than being a bit more verbose.